src/schemas/stoatConfigSchema.json

```diff
@@ -59,6 +63,39 @@
           }
         }
       }
+    },
+    "job_runtime_plugin": {
+      "type": "object",
+      "required": ["job_runtime"],
+      "properties": {
+        "metadata": {
+          "type": "object",
+          "additionalProperties": true
+        },
+        "job_runtime": {
+          "type": "object",
+          "additionalProperties": true,
+          "properties": {
+            "included_jobs": {
+              "type": "object",
+              "properties": {
```

Comment:
Should the size be configurable here? (not blocking this PR ofc)

Reply:
Good point. Will do.

Reply:
Done.